PyISY/Nodes/node.py

```diff
@@ -154,3 +155,24 @@ def dim(self):
             self.parent.parent.log.info('ISY dimmed node: ' + self._id)
             self.update(_change2update_interval)
             return True
+
+    @property
+    def spoken(self):
+        if self._spoken is None:
+            # Get the device notes, currently only the Spoken property is saved.
```

Comment:
Makes sense, but I'll call the method _get_notes, because it is calling getNodeNotes.  Currently it only saves the spoken property, but I might want to add notes in the future.